server/middleware/__tests__/validation.spec.js

```diff
@@ -450,6 +450,18 @@ describe('Validators', () => {
             expect(validators.oneOf({ submittedFormData: { 'delay': 'delay' }, options: options, message: 'test error message' })).toEqual(null);
         });
     });
+
+    describe('Valid within given days validator', () => {
+        it('should reject if more than 30 days after current date', () => {
```

Comment:
Would make the descriptions on these more generic, to avoid an inference it only works for 30 days.

i.e. should reject if more days than the given days allowed

Also apply same for the test on line 460

Reply:
Good shout, updated